src/main/groovy/au/org/ala/auth/PreAuthorise.groovy

```diff
@@ -30,7 +30,11 @@ import java.lang.annotation.Target
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
 public @interface PreAuthorise {
-    String requiredRole() default "ROLE_ADMIN"
+    /***
+     *  A list of roles that is allowed to access the method. The user must have at least one role to access the method.
+     * @return
+     */
+    String[] requiredRoles() default ["ROLE_ADMIN"]
```

Comment:
Nit: requiredRoles implies (to me) that all roles are required.  Suggest a change of name to something like `allowedRoles`